setup.py

```diff
@@ -27,6 +27,11 @@
     package_version = re.compile(
         r".*__version__ = '(.*?)'", re.S).match(v_file.read()).group(1)
 
+
+requirements = [
+    'psutil==4.2.0',  # used to find processes leaked during termination
```

Comment:
set minimum version here.
Strict version pinning should be done in requirements-test.txt

Simialr to https://github.com/ClearcodeHQ/pytest-dbfixtures/blob/master/requirements-test.txt
and https://github.com/ClearcodeHQ/pytest-dbfixtures/blob/master/.travis.yml#L17